### ISSUE TITLE
Preference to disable Git indicators

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#232]
They are quite visually heavy and distract from writing. Would be good to have them optional.

**Comment by astoilkov** [comment]
We have changed the way Git indicators look by making less distracting. Do you think you still need that preference?

**Comment by borekb** [comment]
Much less of a "need" now :)

**Comment by astoilkov** [comment]
We want to make an app that is as clean as possible - which means as few settings as possible. I will close this issue for now but if someone in the future raises the same question I will include you in the conversation and we could consider adding the option.

Is this ok for you?

**Comment by borekb** [comment]
Sure, no problem. I think though that the `Preferences.md` scales very well – it does not clutter the UI and there could be some "advanced" section at the bottom with things like this. But the indicators look nice now.

**Comment by astoilkov** [comment]
Yep. I agree with the advanced section. If we stumble on few options that could go there we could actually create such a section. For now we are on hold on such an idea.

**Comment by schaveyt** [comment]
What is odd is that I am seeing the git gutter even when I am on a file that is not within a git repo. Normally, this would not be a big deal but because I am working on a very large technical document (326K) it make Caret crawl/hang when opening the file. Once the file is loaded, the responsivness of the app returns.

So having the the ability to disable this feature would still be recommended :-)

**Comment by astoilkov** [comment]
Hi @schaveyt 

Can you provide a little more information about your case so we can fix the incorrect git indicators. On what OS are you? Do you have git on your machine? Is git gutter showing on all files that are not under git or just on some?

Are you sure the slowdown is because of the git indicators? 326K document seems pretty big it is possible that Caret is slowing down because of the complexity of the document itself.

Thanks in advance.

**Comment by borekb** [comment]
Also a quick note, not verified properly by me: when I had an edited file with some Git indicators in carret and then committed the file, the indicators were still there. But maybe they would go away after a while, I don't know.

**Comment by schaveyt** [comment]
Sure. Honestly, I should be jumping to conclusions an blaming the git gutter. This could just be my perception. it could be just an outcome of my large file

### Environment

- OS X El Capitan (10.11.6)
- Git software is installed on the system but no repo for that folder

### Step to Replicate

1. Generate or locate a large markdown file ~350KB or (37525 word count)
2. Create an empty folder on your desktop
3. Paste the file in the folder
4. Open the file

### Observed Behavior

- The tool will look like it loaded the file but...the rendering has actually still in progress
    - The initial few paragraphs are loaded.
    - If you scroll the, the scroll will move but nothing is rendered.
....
waiting...only about 60 secs later...
....
Boom! There it is...all the content is shows up...with the gutters.

- If you attempt to edit the file now, as soon as you type your first lett....pause.....hang....15sec later....you see the letter you type.
